# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/moz.build
# Commit: 16c5dd2cfd60
# Full Hash: 16c5dd2cfd600a62c80ca17cfb35c0868e016056
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r ca8139d58d58 -r 16c5dd2cfd60 gfx/vr/moz.build
--- a/gfx/vr/moz.build	Fri Jun 12 21:53:14 2020 +0000
+++ b/gfx/vr/moz.build	Mon Jun 15 18:25:55 2020 +0000
@@ -31,6 +31,7 @@
 
 LOCAL_INCLUDES += [
     '/dom/base',
+    '/dom/canvas',
     '/gfx/layers/d3d11',
     '/gfx/thebes',
 ]